src/components/FormGroup/FormGroup.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { generateRandomString } from '../../shared';
+import styled, { css } from 'styled-components';
+import { variant } from 'styled-system';
+import { Label } from '../Label';
+import { HelperText } from '../HelperText';
+
+/**
+ * Search component properties
+ * Extends html input element attributes
+ * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#additional_attributes
+ */
+export interface FormGroupProps {
+  /**
+   * Form Group label
+   */
+  label?: string;
+  /**
+   * Id
+   */
+  id?: string;
+  /**
+   * Form Group helper text
+   */
+  helperText?: string;
+  /**
+   * Form Group direction
+   */
+  direction?: 'horizontal' | 'vertical';
+}
+
+const changeDirection = () => {
+  return css`
+    ${variant({
+      prop: 'direction',
+      variants: {
+        horizontal: {
+          flexDirection: 'row',
+        },
+        vertical: {
+          flexDirection: 'column',
+        },
+      },
+    })};
+  `;
+};
+
+const FormGroupWrapper = styled.fieldset<FormGroupProps>`
+  display: inline-flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  align-items: flex-start;
+  border: none;
+`;
+
+const FieldGroupComponent = styled.div<FormGroupProps>`
+  display: inline-flex;
+  ${changeDirection};
+  border: none;
+  > * {
+    margin: 4px;
```

Comment:
Ks. ylempi kommentti. Mieluummin tohon ylempään gap: 8px. Näin tulee ylimääräistä marginia komponentin joka reunalle.